sinopia_client/test/api/LDPApi.spec.js

```diff
@@ -253,13 +258,22 @@
       });
     });
     describe('updateBase', function() {
-      it('should call updateBase successfully', function(done) {
-        //uncomment below and update the code to test updateBase
-        //instance.updateBase(function(error) {
-        //  if (error) throw error;
-        //expect().to.be();
-        //});
-        done();
+      it('should call updateBase successfully', function() {
+        var rsrcCtx = new SinopiaServer.SinopiaBaseResourceContext('http://www.w3.org/2000/01/rdf-schema#', 'http://www.w3.org/ns/ldp#')
+        var baseRsrc = new SinopiaServer.SinopiaBaseContainer('', rsrcCtx, ['ldp:Container', 'ldp:BasicContainer'], 'Sinopia LDP Server')
```

Comment:
example objects instantiated from the JS classes autogen'ed from the new swagger spec models.  note similarity to https://github.com/LD4P/sinopia_server/blob/master/fixtures/base-container.jsonld